### PR TITLE
enanble android login

### PR DESCRIPTION
**Pull request by zheyxu**
@gjwgit Please check https://github.com/anusii/solid/issues/43

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml
Comment (gjwgit) [inline android/app/src/main/AndroidManifest.xml:1]: Do all these permissions need to be added?

**Review by gjwgit — CHANGES_REQUESTED** [review]

File: android/app/src/debug/AndroidManifest.xml [hunk 1]
```diff
@@ -4,4 +4,4 @@
          to allow setting breakpoints, to provide hot reload, etc.
     -->
     <uses-permission android:name="android.permission.INTERNET"/>
-</manifest>
+</manifest>
```
Comment (gjwgit) [inline android/app/src/debug/AndroidManifest.xml:7]: Please restore the final newline. Thanks.

File: android/app/src/main/AndroidManifest.xml
Comment (gjwgit) [reply android/app/src/main/AndroidManifest.xml:1]: Testing suggests none of the `uses-permission`  are required and only the first `intent` is required. Why is the rest included?

File: android/app/src/profile/AndroidManifest.xml
Comment (gjwgit) [inline android/app/src/profile/AndroidManifest.xml:1]: Remove **ALL** permissions not required by the app.

File: android/app/src/main/AndroidManifest.xml
Comment (zheyxu) [reply android/app/src/main/AndroidManifest.xml:1]: Remove unnecessary records

File: android/app/src/profile/AndroidManifest.xml
Comment (zheyxu) [reply android/app/src/profile/AndroidManifest.xml:1]: Removed

File: android/app/src/main/AndroidManifest.xml
Comment (gjwgit) [inline android/app/src/main/AndroidManifest.xml:1]: The app is not expected to make calls or send email so please remove those intents. Also, is the enableOnBackInvokedCallback required for our situation? Thanks.

**Review by gjwgit — CHANGES_REQUESTED** [review]

**Comment by gjwgit** [comment]
@zheyxu Please review my requested change and confirm your approval here. Then @NinadBhat or @anushkavidanage can you please review and approve so that @zheyxu can merge this PR. Thanks.

File: android/app/src/main/AndroidManifest.xml
Comment (zheyxu) [reply android/app/src/main/AndroidManifest.xml:1]: Removed unnecessary components.

**Comment by gjwgit** [comment]
Thanks @zheyxu . Test on Android device. Working just fine.

**Review by gjwgit — APPROVED** [review]

**Review by anushkavidanage — APPROVED** [review]
Reviewed and approved

**Pull request merged**